src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -31,7 +30,10 @@ object DplaMapData {
 case class DplaMapData(
                          sourceResource: DplaSourceResource,
                          edmWebResource: EdmWebResource,
-                         oreAggregation: OreAggregation
+                         oreAggregation: OreAggregation,
+                         // Add id property at the root level (following ES schema)
+                         id: ExactlyOne[String],
+                         `@id`: ExactlyOne[URI]
```

Comment:
Is there something that this is called in the DPLA MAP spec that would be more suitable? I don't think @id means anything to anyone aside from Elasticsearch.